decentralized_devstack/provisioning-utils.sh

```diff
@@ -71,7 +71,7 @@ service_exec_management(){
 		edxapp_service_variant=""
 	fi
 	# $edxapp_service_variant has to be unquoted here
-	service_exec "$service" python ./manage.py $edxapp_service_variant "$@"
+	service_exec "$service" python ./manage.py "$edxapp_service_variant" "$@"
```

Comment:
```suggestion
	service_exec "$service" python ./manage.py $edxapp_service_variant "$@"
```
This has to stay unquoted, otherwise it will produce an empty extra argument for non-lms/cms calls. That's why I had wanted to use the array approach earlier.

Reply:
Just pushed commit with this change